### PR TITLE
Add the event metada as knative-channel labels

### PR DESCRIPTION
**Pull request by montaro**
See also #3491

### CODE REVIEW

File: components/event-bus/internal/knative/util/kn-eventing.go [hunk 1]
```diff
@@ -114,8 +115,10 @@ func (k *KnativeLib) GetChannel(name string, namespace string) (*evapisv1alpha1.
 }
 
 // CreateChannel creates a Knative/Eventing channel controlled by the specified provisioner
-func (k *KnativeLib) CreateChannel(provisioner string, name string, namespace string, timeout time.Duration) (*evapisv1alpha1.Channel, error) {
-	c := makeChannel(provisioner, name, namespace)
+func (k *KnativeLib) CreateChannel(provisioner string, name string, namespace string, labels *map[string]string,
+	timeout time.Duration)(*evapisv1alpha1.Channel, error) {
+	//TODO Add labels
```
Comment (marcobebway) [inline components/event-bus/internal/knative/util/kn-eventing.go]: Do we still need this todo? If not, please remove it.

File: components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go [hunk 2]
```diff
@@ -152,7 +152,15 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 		if err != nil && !errors.IsNotFound(err) {
 			return false, err
 		} else if errors.IsNotFound(err) {
-			knativeChannel, err := r.knativeLib.CreateChannel(knativeChannelProvisioner, knativeChannelName, knativeSubsNamespace, timeout)
+
+			//Adding the event-metadata as channel labels
+			knativeChannelLabels := make(map[string]string)
+			knativeChannelLabels["kyma-source-id"] = subscription.SourceID
```
Comment (radufa) [inline components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go]: Could you please add the "kyma-* strings as const like the other "consts"?

Comment (montaro) [reply components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go]: OK

**Comment by radufa** [comment]
"subscription/reconcile_test.go" should be updated too.

File: components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go [hunk 3]
```diff
@@ -20,6 +20,9 @@ const (
 
 	// Finalizer for deleting Knative Subscriptions
 	finalizerName = "subscription.finalizers.kyma-project.io"
+	subscriptionSourceID = "kyma-source-id"
```
Comment (radufa) [inline components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go]: Could you add an empty line after "finalizerName" and before "subscriptionSourceID"

**Pull request merged**